Resources/views/main/index.html.twig

```diff
@@ -21,7 +21,7 @@
 
             <div class="collapse navbar-collapse col-lg-7 col-xl-6" id="navbarNav">
                 <ul class="navbar-nav ml-0 ml-lg-auto p-3 p-lg-0">
-                    {% if not is_granted('ROLE_EXTERNAL') %}
+                    {% if not is_granted('ROLE_PROCESS_VIEW') %}
```

Comment:
I think we should use `is_granted` here (as opposed to `not is_granted`) to be explicit about which roles can see/use the ui elements: In the near future we'll add a new role (`ROLE_BI`) that should neither grants access to view processes nor grants access to create/edit processes, and using `not` here will show the ui elements to users with this new role.

Reply:
Fixed in 30868cb